presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoRpmController.java

```diff
@@ -40,20 +41,24 @@
 import static javax.ws.rs.core.Response.Status.NOT_FOUND;
 import static javax.ws.rs.core.Response.Status.OK;
 import static javax.ws.rs.core.UriBuilder.fromUri;
-import static org.glassfish.jersey.client.JerseyClientBuilder.createClient;
 
-public final class PrestoRpmControlUtils
+public class PrestoRpmController
 {
-    private static final Logger LOGGER = LogManager.getLogger(PrestoRpmControlUtils.class);
-    private static final JerseyClient JERSEY_CLIENT = createClient();
+    private static final Logger LOGGER = LogManager.getLogger(PrestoRpmController.class);
     private static final String LAUNCHER_SCRIPT = "/usr/lib/presto/bin/launcher";
     private static final Path PRESTO_CONFIG_FILE = Paths.get("/etc/presto/config.properties");
     private static final Gson GSON = new Gson();
     private static final int SUBPROCESS_TIMEOUT = 120;
 
-    private PrestoRpmControlUtils() {}
+    private final Client CLIENT;
```

Comment:
It seems like broken convention: CAPITAL_CASE tends to be used for static constants only (`static final`) while it's not `static`. Why not just `client`?